app/experimenter/docs/openapi-schema.json

```diff
@@ -2902,6 +2927,56 @@
           }
         }
       },
+      "NimbusExperimentCSV": {
+        "type": "object",
+        "properties": {
+          "launch_month": {
+            "type": "string",
+            "readOnly": true
+          },
+          "product_area": {
+            "type": "string"
+          },
+          "experiment_name": {
+            "type": "string"
+          },
+          "feature_configs": {
+            "type": "string",
+            "readOnly": true
+          },
```

Comment:
Nit: just to keep this consistent with our list from the spreadsheet, let's move this below `owner`